modules/backend/classes/BackendController.php

```diff
@@ -111,6 +111,14 @@ public static function extend(Closure $callback)
         self::extendableExtendCallback($callback);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function callAction($method, $parameters)
+    {
+        return parent::callAction($method, array_values($parameters));
+    }
```

Comment:
Why was this required? Has Laravel not fixed it on their end?

Reply:
I believe this is due to Laravel is using array destructing and in php 8 array keys should match parameter names during destructing, so 0, 1, etc. are not valid parameters names in this case.